tools/ipvsadm/ipvsadm.c

```diff
@@ -1597,16 +1575,16 @@ static void print_conn_entry(const ipvs_conn_entry_t *conn_entry,
 
 	snprintf(time_str, sizeof(time_str), "%ds", conn_entry->timeout);
 
-	if (!(cname = addrport_to_anyname(conn_entry->in_af, &conn_entry->caddr,
+	if (!(cname = addrport_to_anyname(conn_entry->af, &conn_entry->caddr,
                     ntohs(conn_entry->cport), conn_entry->proto, format)))
 		goto exit;
-	if (!(vname = addrport_to_anyname(conn_entry->in_af, &conn_entry->vaddr,
+	if (!(vname = addrport_to_anyname(conn_entry->af, &conn_entry->vaddr,
                     ntohs(conn_entry->vport), conn_entry->proto, format)))
 		goto exit;
-	if (!(lname = addrport_to_anyname(conn_entry->out_af, &conn_entry->laddr,
+	if (!(lname = addrport_to_anyname(conn_entry->af, &conn_entry->laddr,
                     ntohs(conn_entry->lport), conn_entry->proto, format)))
 		goto exit;
-	if (!(dname = addrport_to_anyname(conn_entry->out_af, &conn_entry->daddr,
+	if (!(dname = addrport_to_anyname(conn_entry->af, &conn_entry->daddr,
                     ntohs(conn_entry->dport), conn_entry->proto, format)))
```

Comment:
In case of NAT64, `out_af` and `in_af`  are different. "ipvs_conn_entry_t " should keep the af of both directions.